recipes/libsigcpp/2.10.x/conanfile.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
```

Reply:
Why should I remove this? https://github.com/conan-io/conan-center-index/blob/master/recipes/libjpeg/all/conanfile.py#L1-L2
Here it is present

Reply:
This is a policy which got adopted after libjpeg got merged. The next time somebody modifies libjpeg these lines need to be removed there as well.

Please install the latest version of the conan-center hook, so you get the warnings locally as well. Reference: https://github.com/conan-io/hooks/pull/94

CCI is currently under maintenance, but the Conan bot will complain about this as well (the Conan bot uses these hooks as well).